### ISSUE TITLE
How to inline imports that are not relative to the project directory?

### ISSUE THREAD

**Issue by kleinfreund** [jakubpawlowicz/clean-css#1048]
### Environment

* clean-css version - `npm ls clean-css`: 4.2.1
* node.js version - `node -v`: 10.10.0
* operating system: Ubuntu 18.04

### Configuration options

```js
const CleanCSS = require('clean-css');

const cleanCss = new CleanCSS({
  rebaseTo: 'src'
})
```

### Input CSS

```css
@import '/css/settings.colors.css';
```

### Actual output CSS

```css
/* empty */
```

### Expected output CSS

```css
/* contents of src/css/settings.colors.css */
```

### Description

My project is structured like below:

```
project/
├─ src/
│  └─ css/
│     ├─ styles.css
│     └─ settings.colors.css
└─ … 
```

At one point, I read in the contents of `src/css/styles.css` which contains nothing but CSS imports as in the input section above. The `project/` directory is where I run my build commands from.

Now I’m really struggling with having CleanCSS find and inline the imported CSS files. The only way I get it to work is prepending the import URI with `src` which I do not want. That is, an import like the follow will be correctly resolved:

```css
@import 'src/css/settings.colors.css';
```

I thought the `rebaseTo` option would help me here, but I don’t know what it actually does. I looked through the source code of CleanCSS to find answers, but didn’t find a solution (for example, why https://github.com/jakubpawlowicz/clean-css/blob/master/lib/reader/rewrite-url.js#L70-L72 uses `from` and `to` inversed compared to [Node.js: path.relative](https://nodejs.org/docs/latest/api/path.html#path_path_relative_from_to).

**TL;DR**: How do I inline imports that are not relative to the project directory but relative to some sub directory of the project directory?

**Comment by kleinfreund** [comment]
Can’t get this to work.
